playbooks/02 - Use Case - CICD/GitHub - Create Issue Comment.json

```diff
@@ -0,0 +1,121 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "GitHub - Create Issue Comment",
+    "aliasName": null,
+    "tag": null,
+    "description": "Creates an issue comment for the specified repository or user on GitHub",
+    "isActive": true,
+    "debug": false,
+    "singleRecordExecution": false,
+    "remoteExecutableFlag": false,
+    "parameters": [
+        "org_name",
+        "repo_name",
+        "cr_number",
+        "commit_message"
+    ],
+    "synchronous": false,
+    "collection": "\/api\/3\/workflow_collections\/4c58fed3-9e16-451a-9528-0be2b71acda4",
+    "versions": [],
+    "triggerStep": "\/api\/3\/workflow_steps\/da0edd7f-6f64-46be-be72-cafe8f4aff07",
+    "steps": [
+        {
+            "@type": "WorkflowStep",
+            "name": "Add Issue Commit",
+            "description": null,
+            "arguments": {
+                "name": "GitHub",
+                "config": "2d753ea1-2fd8-4343-b476-6fd8ba973346",
+                "params": {
+                    "org": "{{vars.input.params['org_name']}}",
+                    "body": "{{vars.input.params['commit_message']}}",
+                    "repo": "{{vars.input.params['repo_name']}}",
+                    "repo_type": "Organization",
+                    "issue_number": "{{vars.input.params['cr_number']}}"
+                },
+                "version": "1.0.1",
+                "connector": "github",
+                "operation": "create_issue_comment",
+                "operationTitle": "Create Issue Commnet",
```

Comment:
This title has a typo. Please correct in the connector info.json as well

Reply:
done